freqtrade/leverage/liquidation_price.py

```diff
@@ -0,0 +1,373 @@
+from typing import Optional
+
+from freqtrade.enums import Collateral, TradingMode
+from freqtrade.exceptions import OperationalException
+
+
+def liquidation_price(
+    exchange_name: str,
+    open_rate: float,   # Entry price of position
+    is_short: bool,
+    leverage: float,
+    trading_mode: TradingMode,
+    mm_ratio: float,
+    collateral: Optional[Collateral] = Collateral.ISOLATED,
+
+    # Binance
+    maintenance_amt: Optional[float] = None,
+
+    # Binance and Gateio
+    wallet_balance: Optional[float] = None,
+    position: Optional[float] = None,   # Absolute value of position size
+
+    # Gateio & Okex
```

Comment:
I would remove both the whitespaces - as well as the comments here. It's duplicate information from the docstring - and makes this not even remotely look like a function header.

Reply:
Sure, are you ok with the side comments? They're still duplicate but there's so many parameters here that personally I find them helpful. 

Also sorry that the comments in this file were so messy before. I've cleaned them up.

Reply:
These comments are fine 👍